src/common/latest/sonarqube/Endpoint.ts

```diff
@@ -75,8 +71,14 @@ export default class Endpoint {
 
     // When proxy is used we use HttpsProxyAgent or HttpProxyAgent to handle it.
     if (proxyUrl) {
-      const AgentClass = endpointUrl.startsWith("https://") ? HttpsProxyAgent : HttpProxyAgent;
-      options.agent = new AgentClass({ proxy: proxyUrl });
+      // We need to set proxy to false to avoid conflicts with agent proxy configuration
+      options.proxy = false;
```

Comment:
refer to https://github.com/coder/vscode-coder/commit/4dc0db909446feaa49490b7b580a1f12461a06a1